test/bigquery.ts

```diff
@@ -213,6 +221,17 @@ describe('BigQuery', () => {
       );
     });
 
+    it('should allow overriding the baseUrl', () => {
+      const baseUrl = 'http://bigquery.test.url.com';
+      setHost(baseUrl);
+
+      bq = new BigQuery();
+      const calledWith = bq.calledWith_[0];
+      assert.strictEqual(calledWith.baseUrl, baseUrl);
+
+      unsetVariables();
```

Comment:
It's possible this line won't be reached. We should cache the value of `process.env.BIGQUERY_EMULATOR_HOST` and restore it in an `after` hook.